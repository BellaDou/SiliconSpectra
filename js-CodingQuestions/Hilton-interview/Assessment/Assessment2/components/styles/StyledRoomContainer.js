import styled from "styled-components";

const StyledRoomContainer = styled.div`
  display: flex;
  flex-direction: row;
  justify-content: start;
  align-items: center;
`;
export default StyledRoomContainer;
