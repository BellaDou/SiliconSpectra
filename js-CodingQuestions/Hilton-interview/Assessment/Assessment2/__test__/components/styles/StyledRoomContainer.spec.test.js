import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import StyledRoomContainer from "../../../components/styles/StyledRoomContainer";

test("Should properly render style component", () => {
  const tree = renderer.create(<StyledRoomContainer />).toJSON();
  expect(tree).toMatchSnapshot();
  expect(tree).toHaveStyleRule("display", "flex");
  expect(tree).toHaveStyleRule("flex-direction", "row");
  expect(tree).toHaveStyleRule("justify-content", "start");
  expect(tree).toHaveStyleRule("align-items", "center");
});
