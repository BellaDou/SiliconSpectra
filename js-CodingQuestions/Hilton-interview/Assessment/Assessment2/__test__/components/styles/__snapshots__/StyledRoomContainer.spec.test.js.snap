// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should properly render style component 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: start;
  -webkit-justify-content: start;
  -ms-flex-pack: start;
  justify-content: start;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<div
  className="c0"
/>
`;
